src/views/v2/month/calendar-body/day/calendar-events/calendar-event/featured-image.php

```diff
@@ -26,17 +26,17 @@
 ?>
 <div class="tribe-events-calendar-month__calendar-event-featured-image-wrapper">
 	<a
-		href="#"
-		title="<?php echo esc_attr( get_the_title( $event->ID ) ); ?>"
+		href="<?php echo esc_url( $event->permalink ); ?>"
+		title="<?php echo esc_attr( $event->title ); ?>"
```

Comment:
We're in the loop when we render the templates, see [the README.md](), if we use `$event->title` we're getting the raw database value, not the filtered event title.
I'd keep using `get_the_title( $event->ID )` here.

Reply:
Fair, good point.